docs/metadata.md

```diff
@@ -104,6 +104,6 @@ and made available in `/run/config/userdata`.
 
 ## HyperKit
 
-HyperKit does not distiguish metadata and userdata, it's simply
+HyperKit does not distniguish metadata and userdata, it's simply
```

Comment:
This spelling is also wrong, "distinguish" is correct.

Reply:
Ha! Fixed one typo with another!